src/cc2olx/main.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import traceback
+
+from cc2olx import filesystem
+from cc2olx import olx
+from cc2olx.cli import parse_args
+from cc2olx.models import Cartridge
+from cc2olx.settings import collect_settings
+
+
+def convert_one_file(settings, input_file):
+    workspace = settings['workspace']
+    filesystem.create_directory(workspace)
+
+    cartridge = Cartridge(input_file, settings)
+    cartridge.load_manifest_extracted()
+    cartridge.normalize()
+
+    xml = olx.OlxExport(cartridge).xml()
+    olx_filename = cartridge.directory.parent / (
+        cartridge.directory.name + "-course.xml"
+    )
+
+    with open(olx_filename, "w") as olxfile:
+        olxfile.write(xml)
+
+    tgz_filename = cartridge.directory.parent / (
```

Comment:
Would it make sense to put this into a differently named folder? `tmp` insinuates to me that it's disposable data, not the primary output of the program.

Maybe we could output into a folder named `output`, or into the current working directory for a single tar.gz file.